assets/src/edit-story/utils/localStore.js

```diff
@@ -32,7 +32,11 @@ function getItemByKey(key) {
 }
 
 function setItemByKey(key, data) {
-  localStorage.setItem(key, JSON.stringify(data));
+  try {
+    localStorage.setItem(key, JSON.stringify(data));
+  } catch {
+    // Do something
```

Comment:
eh... do we need to actually do anything here? should we log an error?

Reply:
I'll think about adding this to our tracking. Don't think it should be surfaced to the user though.